data/projects.json

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var projectsList = [
```

Comment:
Since this is a JSON file, you need to remove everything that is not data.
Having the above lines will actually break your code.